lib/optional_dependency_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+def require_optional_dependency(name)
```

Comment:
Would it be possible to have this gem return `true` if it succeeds?

Reply:
Yep